django-stubs/contrib/auth/backends.pyi

```diff
@@ -33,7 +34,7 @@ class ModelBackend(BaseBackend):
         is_active: bool = ...,
         include_superusers: bool = ...,
         obj: Optional[Model] = ...,
-    ): ...
+    ) -> QuerySet[AbstractBaseUser]: ...
```

Comment:
shouldn't this be with `_AnyUser`?

Reply:
Won't work

> django-stubs/contrib/auth/backends.pyi:37: error: Type argument "Union[AbstractBaseUser, AnonymousUser]" of "_QuerySet" must be a subtype of "Model"

Reply:
This method does a DB query, so it only returns user models -- it can't return `AnonymousUser`.

Similar to how `authenticate()` returns `AbstractBaseUser`.